### ISSUE TITLE
Upgrading Istio using Helm

### ISSUE THREAD

**Issue by ymesika** [istio/istio.github.io#1752]
In the instructions of [Upgrading Istio](https://preliminary.istio.io/docs/setup/kubernetes/upgrading-istio/) I think that users who installed Istio 0.8.0 using Helm should simply be instructed to execute:
```sh
helm upgrade istio install/kubernetes/helm/istio --namespace istio-system
```
to upgrade.

Also upgrade commands seems to be not updated (no more `istio.yaml` in releases, proxy image incorrect, are those the right values to be used?).

**Comment by sdake** [comment]
@rcaballeromx - see https://github.com/istio/istio.github.io/issues/1752#issuecomment-404365076 - but feel free to work on it and resolve it.

Cheers
-steve

**Comment by sdake** [comment]
@ymesika validated upgrade here: https://github.com/istio/istio/pull/7062

**Comment by geeknoid** [comment]
Related to #1798

**Comment by jasmine-jaksic** [comment]
@rcaballeromx Do you have a PR for fixing this?

**Comment by ymesika** [comment]
Fixed by #1883
